### PR TITLE
Symfony 2.8 support

### PR DESCRIPTION
**Pull request by nocive**

### CODE REVIEW

**Comment by nocive** [comment]
Changelog updated, not sure if that's how you wanna have it, so let me know if you require further changes :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ...
 
+## 3.0.0 - 2019-07-02
```
Comment (Jean85) [inline CHANGELOG.md]:
```suggestion
## 3.1.0 - 2019-07-02
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+ - Add support for Symfony 2.8.
```
Comment (Jean85) [inline CHANGELOG.md]: I'll take advantage of your PR to backfill this:
```suggestion
 - Add support for Symfony 2.8 (#233, thanks to @nocive)
 - Fix handling of ESI requests (#213, thanks to @franmomu)
```

**Comment by nocive** [comment]
done & dusted :rocket:

**Comment by Jean85** [comment]
Thank you @nocive! I will tag it now!

**Pull request merged**